db/version_set.cc

```diff
@@ -3124,6 +3182,7 @@ void VersionStorageInfo::CalculateBaseBytes(const ImmutableCFOptions& ioptions,
       }
     }
   } else {
+    dynamic_level_bytes_ = true;
```

Comment:
can we set it on the constructor?

Reply:
Yes, before I just wanted to keep the changes small. I'll change it